app/pages/index/ui.test.js

```diff
@@ -1,5 +1,5 @@
 import nock from 'nock';
-import { ClientFunction, Selector } from 'testcafe';
+import { ClientFunction/* , Selector */ } from 'testcafe';
```

Comment:
do we still need the `pages/index` folder?

Reply:
I was going to repurpose this whole folder as dashboard

Reply:
ok 👍